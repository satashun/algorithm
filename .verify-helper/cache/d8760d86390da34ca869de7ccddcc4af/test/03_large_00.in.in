0001000101101000000100000100000110001110011000011010010000000000001000010010100000000001101000001000100101000000000000011001000110001110000001000001101010000100000000000010000000110000010101010000010000000001110010000011010110010000100101000100010000100000001000100000001001001110010110000001000110010010001001100000111001100000011000100000000000000010101000000010000000000000000010000010000010011100101000000101110101000000000010101000000001000000010010000100010001001000000001010000000000000010001100000101011000000010000000000100000001000001000010011000001011000001000000010000100010000010010000001100001000000000000010111000100000001001101000000001100000100100000100000100011100100000000000001000010010000010001001011000001000010100101100010000000100000001010000000001100010000001100100101101000000100000000000000000001000010001110101100010100000000000100100000000100010001000010110000010001100000010011000100000100001000011001010011010101001000010001001010000000000000010100000000110100000000100000000000000000100010000000000010000100000000000010000000100010000000000000000000000000000000000000001001101000010000010000100000000000000000010000000000000100000000000000000010000000000000000000000000000000000010000000000000000000000010000000000000000000010001000000000000001000000000000001000000010000000000000000000000010010000000000010000000000001000010011000010000000001000000001010100000000000000000000000000000000000000000000000100001000000000000000000000000000000100000010000010000010000000000000001000100000000010000000000010000000000000000000100000000000000010000000000000000000000001000000000000000000000000000010000000000000000000000000001000000000000000000000000000010010100010000000010000000000000000001000100000010000000000000000000000000100000001000000000000000000000000000001000100010000010000000000000010010000000001000000000010000100000000000001001000000000000000000000100000000001000000000000000110000000001000000000000000000001000000000000000000000000000000000000000000000000010000000000001001000000000000000000000100000010000001000000000010000000100000000000000001000000001010000000000000000010000010000000100000000000000001000000000000000100000000000000000000000000000000000001000000000000100000000000100100000000101000000000000000000000001000000000001100000001000000000000000001001000000000100010001000000000001000100000100000000000000000000010000000100100001000000010000000000000010000100001100000000001000000000001000000100010000000000000001100001000000111000010000000000000010010000001010000001100000000010000010000000010100000000000000000000000000100100000000000000000000000000100000000000110100000000000000000000000000000000001000000000000000000000001010000000000000000000000000000000000000000000010000100000000000000000000000000000000000000000001001010100000000000001000000000000000000000000000100000000000000100000000000001000000000000000001000000000100000100000000000000000000001000001000000000100000001000000000000000001000000000000100001000000000000000001000000001000000000000010100000001000000000000000000000000000000000000000000000000000001000000000100001000010000000000001000000000100000000000000010000000000000010000011000000000000000100000000000000000000000000100010010100000001000000000000001100000000000000000000000000010000000010000000000000010000000000000000000000010000000000000000000000000011000000000000000000000000100000000000000000000001010010000000000000000000000000000000000000000000000000000000000100000000100011000000000010000001000000000000000000000000000000001000000000000000000100100100100100000000000000101010000000000000000000000010100000010000000000000000000001000000000000000000000000000001000000000010000010000001100000000000100000000000001000000000000000000010010000000001000000000000000000000001000000000100000000000000100000000000010000010000010000000100000001000000100000000000000000000000000000000000000000000000001000100000000000001000000000000000000100000000000000000010000000100000000000000000000000000000000000000000110000000000000100000000000000000000000001000000000000000000100010000000000000000000000010010000000000010000000010000000000000001000000000000000000100000000010100000010000000000000000000000000000000000000000010000000010000000000000000000000000000000000000000000000000000000000100000000001000000000000000000000000000100000000000000000001000000000000000110100011000100001100000000000100000100000100000010000000000000100000100000000000000000000000000000010000000000000000000000000000110000000100000010000000000000000000010000000000000110000100000000000101000000000000000100000000000000000010000000000000000000000000000000000000000000100000000000010000000000000001100001000000000001000000000001010000000000001010000000000000000000010000000000001000000000000000000000100000001000001000010100000000000000000000000000100000010000000000000000000100000000000000000000000000000001000000000000000001000000010001000000000000000000000000000000000000010000100000010000000100010000000000000000000000000000000000100000000000000000000000000000111000000000000000000000000000000001100000010000010000000000001000000000000110000000000000000000110000000000100010100000000000000010000000000000000000000000000001000101000000000000000000000010000000000000000000000000000010000010000000000000000000000000000000000000000000001000000000000001000000000000000000100000000000000010100010010000000000100000000000000000000000000000000000000000010000000001000000000001000010000000000000000000000000000000000000000010010001000000000000000000100001000100000000001110001000100000000000000000000011000000000000000000000001000000000000000001000000100000000000000001001010000000000000000000000000000000000010000000000000010000000001000010000000000000100010000000000000000000000000000000000000001000000010000000000000000001000001000000100000000001000000000000000000001000000000000001000000000000000010000010000000000010100000000000000000000001000100000000011000000000100000000001000100000000000010000000000000000000000000000100001000001000000001000000000000000000000000000000010000010001000000000000000000000000000011000000000110000000000000001000001000000000000000010000000100000000100000000000000000000000000010000100000000000000000010000000000010000000000000000000000010000000100000000000010000110000000000000000000000000010000000100000000000000110000010000000000100000001000100000000000000000000000000000010000000000000000000011100000000000101000000000000000000000000000000000000001000000000000000000000000000010000000011000000000000000000000011000000000000000000001001000000000000000000100000000000000000000010000000000000000100000010000000000000000010100000000000000000000000000100000000000000000101100100000000000000000000001000000000000000100101000000000010001011000000000000000000000000110000000000000000000000011000000001000000000000000000000000010100000000000001000001001000000000000010000000000000000000000000010000000000000001000000000000000000000000000001000001000011000100000000000000000000000000000000001001000001000000000100000000000000100001010000000000001000000001000000010000000001000000000000000000000010000000000000010000000000000010000010000000000000000000100000000000000000000001000000100000000000000000001000000000101000000000000000000010000000000000000000000000101000000000000000000000000000000000001000100000000000000010000000000000100000000010000000000010000000000000000000000000000000000010000000000000000010000000000000000100010000000000000000001000000001000000000101000000000010000000000000000000000000000100000000000000000000000000000100000000000000000001000000000100000000000010000100000110000000101100000010000000000000001010100000000000000100010010000000000000001000000000010000110000000000000010000000010001001000000000000000100000000000010000000000010000001000000010000000100010000000000000000000000000000000000000000011000001000000000000001000000100001000000100100100100010000100000000000101000000000100000000000010000000000100100000000001000000010010000000000000000000000000000000000000000000000000000000110000000000000000010000000000000000000110000000000001000010000000000000000100000000000000000001000000000100000000000000000001000000000000000000000000000010000100000000101000000000000000000000000100000000000000000000000100000110000010001000000000000000000000000010010000010000000000000000000000000000000000000000000000100000000000000000001010000000000000000000000000011100001010000000000001000000000000010000000000010000000000000000000000000100000000011000000000000000000000000000000010000001000000100000000000000000000000001000110000000000010000000000000100000000000000000000010000001000110000010010000000100000000100000000000000000000000000000000010000010000001010000000000000000000000000000100100000000100000100000000000000000000000000000000000000010000000000000000000000000000000000000000000000100000000100000000000000000000000000000010011100000000000100000000000000000000000010000100000000000000000000000000000000000000000000000000000000000000000010000000010000001000000000000001100000100000000000000001000000000000000000000000000000000000000000000000000000111000000010000100001000000000000001000000010000000001010000000100100100000000000010000000010001000000000000000000100000010000000000000010000000000000000000001000000000100000100000000000010000000000000100010000000000000000010000000000000000010010001000000010000100000000000000000000010000001100000100001010001000001000000001001000000000000010100000100000010100000000000000000010000000100000000000000001000000000000000000000000000000000000000000000000000000010000000000000101001000000000000000000110000000010010001000000010100000001000000000000000000001000101000000101000000000000000000010000000000000000000100000000000000100001000010110000000000000010011000000000000000000000001000000000000000000110001001000000000000010110000001000000000001000000000000000000000100000000000000000000000001000100000000000000000000000000000000000000000000000000000000000000001000000000000000010000010100000000000000000000000000100000100000000000000000000000000000000000
00110
