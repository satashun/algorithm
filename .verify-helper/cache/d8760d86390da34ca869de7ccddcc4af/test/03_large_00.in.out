29
39
45
85
117
125
145
176
216
293
307
319
327
496
549
598
637
649
769
781
892
903
924
933
983
1102
1356
1913
2272
2413
2464
2518
2615
3153
3229
3333
3459
3670
3983
4334
4341
4351
4463
4514
4642
5044
5084
5105
5541
5898
6057
6068
6239
6290
6465
6489
6737
6762
6921
7562
7655
7797
7985
8024
8210
8436
8513
8573
8968
9337
9542
9699
9743
